src/rollover.py

```diff
@@ -352,5 +333,94 @@ def main():
         print "Rollover complete!"
 
 
+def main():
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers()
+
+    #
+    # Rollover args
+    #
+    rollover_parser = subparsers.add_parser('rollover',
+                                            help="rollover ECS nodes")
+    rollover_parser.set_defaults(func=main_rollover, scale_down=False)
+
+    rollover_parser.add_argument('-t',
+                                 '--timeout',
+                                 type=int,
+                                 default=30,
+                                 help="`docker stop` timeout")
+    rollover_parser.add_argument('-n',
+                                 '--noop',
```

Comment:
I would prefer to name this `--dry-run` instead of `--noop` to make it more clear it doesn't actually apply changes "in production," whereas noop sounds like it does nothing and is more equivalent to `pass`.